src/shutdown/detach-dm.c

```diff
@@ -99,23 +99,31 @@ static int delete_dm(DeviceMapper *m) {
         assert(major(m->devnum) != 0);
```

Comment:
> fix systemd-shutdown hang all the time

The commit message sound slightly overclaim, and less information. Could you explain more?